addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -243,54 +243,86 @@
+        <div class="content-slide-infos ml-2">
+            <i t-if="channel.can_publish" class="fa fa-arrows mr-2 text-muted"></i>
+            <i t-if="slide.slide_type == 'document'" class="fa fa-file-pdf-o mr-2 text-muted"></i>
+            <i t-if="slide.slide_type == 'infographic'" class="fa fa-file-picture-o mr-2 text-muted"></i>
+            <i t-if="slide.slide_type == 'video'" class="fa fa-play-circle mr-2 text-muted"></i>
+            <i t-if="slide.slide_type == 'link'" class="fa fa-file-code-o mr-2 text-muted"></i>
+            <i t-if="slide.slide_type == 'webpage'" class="fa fa-file-text mr-2 text-muted"></i>
+            <i t-if="slide.slide_type == 'quiz'" class="fa fa-question-circle mr-2 text-muted"></i>
+            <i t-if="slide.slide_type == 'certification'" class="fa fa-trophy mr-2 text-muted"></i>
+            <a class="link-to-slide" t-attf-href="/slides/slide/#{slug(slide)}"><span t-field="slide.name"/></a>
+        </div>
+        <div class="content-slide-controls mr-2">
+            <i t-if="not slide.id in user_progress or not user_progress[slide.id].completed" class="check-done fa fa-check-circle text-muted mr-1"></i>
+            <i t-if="slide.id in user_progress and user_progress[slide.id].completed" class="check-done text-primary fa fa-check-circle mr-1"></i>
+            <a t-if="channel.user_id == user and not slide.slide_type == 'webpage'" t-attf-href="/web#id=#{slide.id}&amp;action=#{slide_action}&amp;model=slide.slide&amp;view_type=form"><i class="fa fa-pencil text-muted mr-1"></i></a>
+            <a t-if="channel.user_id == user and slide.slide_type == 'webpage'" t-attf-href="/slides/slide/#{slug(slide)}?enable_editor=1"><i class="fa fa-pencil text-muted mr-1"></i></a>
+            <i t-if="channel.user_id == user" t-attf-slide_id="#{slide.id}" class="fa fa-trash text-muted oe_slide_js_delete_slide"></i>
+        </div>
+    </li>
+</template>
+
+<template id="course_content_training_category" name="Category template for a training channel">
+    <div style="width:50%;" class="d-flex align-items-center">
+        <i t-if="channel.can_publish and category" class="fa fa-arrows mr-3 text-muted"></i>
+        <t t-if="category">
+            <span t-field="category.name"/>
+        </t>
+        <t t-else="">
+            <span>Uncategorized</span>
+        </t>
+    </div>
+    <a  class="mr-2 oe_slide_js_upload"
+        role="button"
+        aria-label="Upload Presentation"
+        href="#"
+        style="font-size: 1.5rem;text-decoration: none;"
+        t-att-data-channel-id="channel.id"
+        t-att-data-category-id="category.id if category else None"
+        t-att-data-can-upload="channel.can_upload"
+        t-att-data-can-publish="channel.can_publish">+</a>
+</template>
+
 <template id="course_content_training" name="Content of a training channel">
```

Comment:
course_slides_list

Reply:
Done but "website_slides.course_content_documentation" should probably be cleaned too, I leave it to you.